woocommerce-services.php

```diff
@@ -819,6 +823,72 @@ public function deactivate_notice() {
 			<?php
 		}
 
+		public function check_jetpack_install() {
+			if ( defined( 'JETPACK_DEV_DEBUG' ) ) {
+				return true;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
+				return false;
+			}
+
+			$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
+			return $user_token && is_object( $user_token ) && isset( $user_token->external_user_id );
+		}
+
+		public function admin_jetpack_notice() {
+			wp_enqueue_style( 'wc_connect_banner' ); // Including the CSS this early will mostly prevent a flash of Core styles in the banner
+			add_action( 'admin_notices', array( $this, 'show_jetpack_notice' ) );
+		}
+
+		public function show_jetpack_notice() {
+			$button_class = 'button-primary';
+
+			if ( class_exists( 'Jetpack_Data' ) ) {
+				$screen = get_current_screen();
+				if ( 'jetpack' === $screen->parent_base || 'plugins' === $screen->base ) {
+					return; // Jetpack Dashboard and the plugins list screen already have big "Connect to WordPress.com" buttons
+				}
+
+				$notice_text = __( 'To get started, please connect Jetpack to your WordPress.com account.', 'woocommerce-services' );
+				$button_label = __( 'Connect to WordPress.com', 'woocommerce-services' );
+				$redirect_to = admin_url( 'admin.php?page=wc-settings&tab=shipping' );
+				$button_url = Jetpack::init()->build_connect_url( false, $redirect_to, 'woocommerce-services' );
+			} else {
+				$activate_url = wp_nonce_url( 'plugins.php?action=activate&plugin=jetpack/jetpack.php', 'activate-plugin_jetpack/jetpack.php' );
+
+				if ( 0 === validate_plugin( 'jetpack/jetpack.php' ) ) {
+					$notice_text = __( 'To get started you need to activate Jetpack.', 'woocommerce-services' );
+					$button_label = __( 'Activate Jetpack', 'woocommerce-services' );
+					$button_url = $activate_url;
+				} else {
+					$notice_text = __( 'To get started you need to install Jetpack.', 'woocommerce-services' );
+					$button_label = __( 'Install Jetpack', 'woocommerce-services' );
+					$button_url = '#';
+					$button_class .= ' wcc-install-jetpack';
+
+					wp_enqueue_script( 'updates' );
```

Comment:
You can define `'updates'` as a dependency for `wc_connect_banner` in the call to `wp_register_script()`.